media-libs/agar/agar-1.5.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A modern open-source, cross-platform toolkit for graphical applications"
+DESCRIPTION="Cross-platform GUI toolkit.  Agar provides a base framework and \
+a collection of GUI widgets from which GUI applications can be built.  Agar \
+can be styled and themed using a CSS-style engine."
+HOMEPAGE="http://libagar.org/index.html.en"
+COPYRIGHT="2002-2016 Hypertrition, Inc."
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="http://stable.hypertriton.com/agar/agar-1.5.0.tar.gz"
+CHECKSUM_SHA256="82342ded342c578141984befe9318f3d376176e5f427ae3278f8985f26663c00"
+
+ARCHITECTURES="!x86_gcc2 !x86 !x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 !x86"
+
+PROVIDES="
+	cmd:agar
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:fontconfig
+	lib:libfreetype$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtool$secondaryArchSuffix
```

Comment:
But it doesn't use libtoolize, it just uses libtool.  This is using bsdbuild instead of the standard gnu autotools.  I left it out and was getting a libtool missing error, so I added it in and the error went away.